fr/controllers.rst

```diff
@@ -255,7 +255,7 @@ vue ou un élément dont le chemin est relatif au dossier **src/Template**.
 Cela permet un affichage direct des éléments, ce qui est très pratique lors
 d'appels AJAX::
 
-    // Rend un élément dans src/Template/Element/ajaxreturn.ctp
+    // Rend un élément dans templates/element/ajaxreturn.php
     $this->render('/Element/ajaxreturn');
```

Comment:
`/element/ajaxreturn`

Reply:
Change path in `$this->render('/element/ajaxreturn')`, but leave full path in comment.